test/index.html

```diff
@@ -229,15 +229,15 @@
         assert.equal(data.frameworks["Flow"].firstUsed, data.frameworks["Flow"].lastUsed);
         assert.closeTo(data.frameworks["Flow"].firstUsed, new Date().getTime(), 10000);
       });
-      test('valo-detected', function () {
+      test('lumo-detected', function () {
         window.Vaadin = window.Vaadin || {};
-        window.Vaadin.Valo = { version: "2.0.0-alpha5" };
+        window.Vaadin.Lumo = { version: "2.0.0-alpha5" };
         const stats = new UsageStatistics();
         stats.gatherer.gather(stats.storage);
         const data = stats.storage.read();
-        assert.equal(data.themes["Valo"].version, "2.0.0-alpha5");
-        assert.equal(data.themes["Valo"].firstUsed, data.themes["Valo"].lastUsed);
-        assert.closeTo(data.themes["Valo"].firstUsed, new Date().getTime(), 10000);
+        assert.equal(data.themes["Lumo"].version, "2.0.0-alpha5");
```

Comment:
The first version that we’ll have for Lumo styles is 1.0.0-alpha1 (current is 1.0.0-alpha2).

Reply:
Sure, doesn't matter though as this is just a test